llvm/include/llvm/Transforms/IPO/FunctionSpecialization.h

```diff
@@ -131,13 +131,16 @@ struct Spec {
   // Profitability of the specialization.
   unsigned Score;
 
+  // Cost of the specialization, in terms of codesize.
+  unsigned CodeSizeCost;
```

Comment:
If we decide to keep this field please s/CodeSizeCost/CodeSize and update the comment please ("Number of instructions of the specialization" or similar)

Reply:
Done